package.json

```
@@ -73,6 +73,7 @@
     "@leafygreen-ui/checkbox": "6.0.6",
     "@leafygreen-ui/code": "9.0.0",
     "@leafygreen-ui/confirmation-modal": "2.2.2",
+    "@leafygreen-ui/expandable-card": "^0.9.1",
```

We generally try to pin dependencies to specific versions
```suggestion
    "@leafygreen-ui/expandable-card": "0.9.1",
```